docs/about_PSReadLine.help.txt

```diff
@@ -1,179 +1,179 @@
-
-        These two functions retrieve useful information about the current state of
-        the input buffer.  The first is more commonly used for simple cases.  The
-        second is used if your binding is doing something more advanced with the Ast.
-
-    IEnumerable[Microsoft.PowerShell.KeyHandler] GetKeyHandlers(bool includeBound, bool includeUnbound)
-
-        This function is used by Get-PSReadLineKeyHandler and probably isn't useful in a custom
-        key binding.
-
-    Microsoft.PowerShell.PSConsoleReadLineOptions GetOptions()
-
-        This function is used by Get-PSReadLineOption and probably isn't too useful in a custom
-        key binding.
-
-    void GetSelectionState([ref] int start, [ref] int length)
-
-        If there is no selection on the command line, -1 will be returned in both start and length.
-        If there is a selection on the command line, the start and length of the selection are returned.
-
-    void Insert(char c)
-    void Insert(string s)
-
-        Insert a character or string at the cursor.  This operation supports undo/redo.
-
-    string ReadLine(runspace remoteRunspace, System.Management.Automation.EngineIntrinsics engineIntrinsics)
-
-        This is the main entry point to PSReadLine. It does not support recursion, so is not useful
-        in a custom key binding.
-
-    void RemoveKeyHandler(string[] key)
-
-        This function is used by Remove-PSReadLineKeyHandler and probably isn't too useful in a
-        custom key binding.
-
-    void Replace(int start, int length, string replacement)
-
-        Replace some of the input.  This operation supports undo/redo.
-        This is preferred over Delete followed by Insert because it is treated as a single action
-        for undo.
-
-    void SetCursorPosition(int cursor)
-
-        Move the cursor to the given offset.  Cursor movement is not tracked for undo.
-
-    void SetOptions(Microsoft.PowerShell.SetPSReadLineOption options)
-
-        This function is a helper method used by the cmdlet Set-PSReadLineOption, but might be
-        useful to a custom key binding that wants to temporarily change a setting.
-
-    bool TryGetArgAsInt(System.Object arg, [ref] int numericArg, int defaultNumericArg)
-
-        This helper method is used for custom bindings that honor DigitArgument.  A typical call
-        looks like:
-
-            [int]$numericArg = 0
-            [Microsoft.PowerShell.PSConsoleReadLine]::TryGetArgAsInt($arg, [ref]$numericArg, 1)
-
-POWERSHELL COMPATIBILITY
-
-    PSReadLine requires PowerShell version 3 or greater and the console host.  It
-    will not work in the ISE.
-
-FEEDBACK
-
-    https://github.com/lzybkr/PSReadLine
-
-CONTRIBUTING TO PSREADLINE
-
-    Feel free to submit a pull request or submit feedback on the github page.
-
-SEE ALSO
-
-    PSReadLine is heavily influenced by the GNU readline library:
-
-        http://tiswww.case.edu/php/chet/readline/rltop.html
+TOPIC

```

Comment:
Did line endings change?

Reply:
Line ending change was already reverted. You may be looking at an old iteration.